tests/lava/lib/optimization/apps/schduler/test_problems.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
Is it really copyright in 2021?
Should be 2023 now.

Reply:
Time machine! Yes.